### PR TITLE
Added Best Matches to GUI

### LINKED ISSUE #12: Display some number of best matches in GUI
Alongside the entire corpus displaying a heatmap, there should also be a "leaderboard" that lists the top 5 or 10 most closely matching words for that query.

### PR DESCRIPTION
**Pull request by Djhranicky**
Added display showing the words with the best scores. Fixed the command line script and added that as well

image
image

### CODE REVIEW

**Comment by TBean18** [comment]
image
Looks like Armin wanted the top 10 matches on the right side. I'll try to make those changes now otherwise, looks great!

**Comment by TBean18** [comment]
image

Looks like we need to fix the match output formatting and maybe truncate some of the float values

**Review by Armintron — COMMENTED** [review]
Neat! Is `RelativeLayout.java` necessary though? I thought Swing already is pretty good about resizing and alignment and stuff, especially with how @TBean18 has been making each componenet compartmentalized.

**Comment by Djhranicky** [comment]
I think since it is left to right it is a lot easier to organize it.  This biggest issue was that the best matches pane kept getting squished to be one line, and then would shrink even further after executing a search. RelativeLayout let me pre-define its size to avoid that issue.

File: src/gui/BestMatchesPanel.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    public static void setBestMatchesContents(Iterator<WordScoreEntry> iterator) {
+        StringBuilder sb = new StringBuilder();
+        for (int i = 0; i < 10; i++) {
+            WordScoreEntry cur = iterator.next();
+            sb.append(String.format("%d:\t%s\t%.2f%n", i + 1, cur.getWord(), cur.getScore()));
+        }
+        bestMatchesPane.setText(sb.toString());
+        return;
+    }
```
Comment (TBean18) [inline src/gui/BestMatchesPanel.java:54]: image
I switched up how we are outputting the best matches to something a lot more simple. Hopefully, it should allow us to perfect the formatting going forward

Comment (Armintron) [reply src/gui/BestMatchesPanel.java:54]: > image I switched up how we are outputting the best matches to something a lot more simple. Hopefully, it should allow us to perfect the formatting going forward
👍

**Review by Armintron — APPROVED** [review]
Lgtm! Great work

**Pull request merged**